composite/action.yml

```diff
@@ -238,6 +239,14 @@ runs:
         LOG_LEVEL: ${{ inputs.log_level }}
       shell: bash
 
+    - name: Save PIP packages cache
+      uses: actions/cache/save@v3
+      if: always()
```

Comment:
with this, you can essentially remove the date trick that accounts for https://github.com/EnricoMi/publish-unit-test-result-action/pull/418#discussion_r1135436316, right?